src/main/java/com/amplitude/HttpCall.java

```diff
@@ -0,0 +1,80 @@
+package com.amplitude;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+
+import javax.net.ssl.HttpsURLConnection;
+import java.io.*;
```

Comment:
Is `java.io.*` generated automatically? I wonder if we should just import the necessary one rather than using the `*` and import all under io.

Reply:
It is, IntelliJ does it.

Reply:
Yes, it's generated automatically. Just thinking that might import lots of things we don't need.
I removed the auto-generated import and import the things we need.